AMM/project/libraries/Forc.toml

```diff
@@ -3,5 +3,3 @@ authors = ["Fuel Labs <contact@fuel.sh>"]
 entry = "interface.sw"
 license = "Apache-2.0"
 name = "libraries"
-
-[dependencies]
```

Comment:
Hmm, convention change. We used to keep these even if there are no dependencies

Reply:
This is spammy, there was no convention. No one removed them before so I've started to if I've happen to see it